modules/serverless-create-kubeconfig-file.adoc

```diff
@@ -0,0 +1,18 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="create-kubeconfig-file_{contect}"]
+= Create a `.kubeconfig` file
+
+Use kubeconfig files to organize information about clusters, users, namespaces, and authentication mechanisms. The CLI tool uses `kubeconfig` files to communicate with the API server of a cluster.
+
+Create a basic `.kubeconfig` file from client certificates.
```

Comment:
Even though this is a very very simple procedure, you should still make this in the format of a procedure module. So with the ".Procedure" heading and an unordered bullet for a single step.